src/Widgets/calendar/ControlHeader.vala

```diff
@@ -19,37 +19,64 @@
 
 
 namespace DateTime.Widgets {
-    public class ControlHeader : Gtk.Box {
+    public class ControlHeader : Gtk.Grid {
         public signal void left_clicked ();
         public signal void right_clicked ();
         public signal void center_clicked ();
+
         public ControlHeader () {
             Object (orientation : Gtk.Orientation.HORIZONTAL);
+        }
+
+        construct {
+            var provider = new Gtk.CssProvider ();
+
+            var label = new Gtk.Label (new GLib.DateTime.now_local ().format (_("%OB, %Y")));
+            label.get_style_context ().add_class ("header-label");
+            label.xalign = 0;
+            label.width_chars = 13;
+
+            try {
```

Comment:
You don't need this try/catch anymore since you're loading from resource

Reply:
Better safe than sorry. What if the resource is lost?

Reply:
It's compiled into the binary, so I guess if part of your binary is missing you have bigger problems :p Anyways, `load_from_resource` doesn't throw an error so this code does nothing: https://valadoc.org/gtk+-3.0/Gtk.CssProvider.load_from_resource.html